### PR TITLE
[MIRROR] Mission Objective Powergame Brig (box station brig rework)

### PR DESCRIPTION
**Pull request by SkyratBot**
Original PR: https://github.com/Citadel-Station-13/Citadel-Station-13/pull/11472
--------------------

## About The Pull Request

TLDR box station brig is no longer a hallway with rooms branching off of it, more of a communal hub where everyone can gather, making it much more !!fun!! to sabotage and generally more of a space for the secoffs to hang out in, rather than them all fucking off in their own corners of the brig, along with adding a couple of new features.

YwmYGWa

First thing that was changed, and originally what this PR was supposed to be but quickly spiraled into a full on brig rework, was the addition of a proper brig infirmary. It always really fucking annoyed me how there was a medbay and surgery tools in the execution room, however there was no actual proper infirmary, this moves those surgery tools out to the brig for public use and in turn gives security a proper little place for them to do surgery. Powercreep? Yes. Making it way less annoying than doing cavity surgeries on the floor of the execution room? Absolutely double yes.

The interrogation room has been put off to the side, specifically on the exterior where a collaborating traitor buddy might be able to blow you out in some sort of extravagant escapade while security is interrogating you, I dunno.

The protolathe has been moved out into a much more communal space, along with the two consoles in the security office for public, and most importantly detective use. The detective can print out his rubber shots but can't actually enter the security office at round start where he's supposed to be able to, also the idea of secoffs printing out their weapons and gear in public space was cool, so yeah fuck it.

EVA has been moved out of brig control and put into its very own little compartment, you will still need the warden to open up the door for you, but you no longer have to enter his gay baby jail zone when ever you need to kill some space fish.

Prisoner cells had to be made very slightly smaller for the incoming changes, more specifically making room for other things, like the new brig control.

szNVQjc

The armory has been completely and utterly revamped, now sporting three different access points into the big bad gun zone, reducing the amount of effort it takes to raid one of the most attractive places on the station, but subsequently makes it much more of a focal point to the brig, the warden especially having a good view to all the shutters from his brand new role as the desk jockey. To compensate for the potential ability to be raided, Armsky has been put on staff to try and reduce any rushes from emaggers, along with some decorative choices like the rubbershot being put in to their very own nonlethal ammunition locker.

Moving onto what I was refering to about the warden getting a revamp, he now sits nicely at the very front of the bridge, tons of buttons to control the flow of the airlocks and the shutters all alround the brig, although he still has to get up if he wants to open up the armory, not that he has to run very far away.

mQbJiTL

And last but not least is the very slightly reworked security office, the gear room has been prettied up and been given a crate to store all of them formal clothing pieces, ask the warden if you would like some of those pretty clothes.

The protolathe has been replaced with a microwave, some donk pockets, and the Zero Sponsored ™️ casting couch, please do not use it for ERP, because that would make me very sad.

And to close it out, the HoS office got some touching up, and has a brand new mascot for the brig, Colonel Chomps, your cutest and totally friendly secbat.

## Why It's Good For The Game

SOUL BAD NEW GOOD

Also box station is as old as citadel itself essentially, a breath of fresh air is usually not a bad thing at all, unless people turn out to really hate it.

## Changelog
:cl:
add: Box Station has gotten a brand new brig. Go and check it out and discover all the quirky little soulbits.
/:cl:

### CODE REVIEW

**Comment by PersianXerxes** [comment]
These changes should set off huge alarm bells for anyone interested in Security players roleplaying effectively or the game being balanced in general.

- Security having a full featured, functioning operating room is a horrible idea (especially when the justification for it was to make it easier to metagame). IF Security needs an operation performed, they should be involving Medical staff regardless of how easy surgery is in our current mechanics, that's good roleplay.

- The wall to the perma wing should remain a solid wall, rather than being fully replaced with windows. As is, these windows allow Security officers to sit in the brig and maintain perfect vision of the perma wing, while also allowing non-Security crew vision into it.

- Moving the sec lathe to the common area makes it accessible to the general public. Reinforced windows are not a barrier and it would have made more sense to set the Security Office doorway to accept general security access, so that it could be accessible to the detective while remaining secure from random crew.

- Putting a Beepsky in the armory gives Security a free alarm anytime it's accessed by unauthorized personnel, which is unbalanced from the mechanical aspect of the game. A warden or AI has to take the time to announce if they witness a break-in, and have to make that announcement at the expense of actually responding to it. Beepsky will automatically announce the break in **while** it attacks the intruder.

These changes are overall only good for powergaming Security and never should have been considered by any sane and reasonable person.
